ci/build.sh

```diff
@@ -4,12 +4,55 @@ set -e -u -x -o pipefail
 
 cd "$(dirname "$0")/.."
 
+if [[ -z ${BUILDKITE+x} ]]; then
+  echo "This script is intended for internal Improbable use."
+  exit 0
```

Comment:
I'd argue that this is an error and should therefore return a non-zero exit code.

Reply:
Yep, changed.